tests/test_clone.py

```diff
@@ -303,6 +304,32 @@ def test_clone_page_no_components(seed_dynamic_data, _db):
 page_id = uuid4()
 
 
+def test_fix_clone_default_pages():
+    # DISCLAIMER: Written on the train when I'd been up since 5am so may not be the most efficient/best approach!
```

Comment:
I guess you could clone a form here instead and then make sure that each page in the cloned form points to a cloned page id rather than the original?
If you just want to test the actual function here (probs what your going for tbf), I think it might be improved by specifying the original template pages and then the fixed cloned pages like so:

`
    original_pages = ..
`
`
    cloned_pages = .. (shows the error in default_page_id)
`
`
    fixed_pages = ..
`

Reply:
Thanks, was going for the second option so have refactored, hopefully clearer!